addon/types/table.js

```diff
@@ -51,6 +51,17 @@ export default EmberObject.extend({
   */
   hooks: {},
 
+
+  appliedFilters: computed('columns.@each.filters.[]', function() {
```

Comment:
I don't see where you use this method ?

Reply:
it's used in the app using the table, i first defined it there but i think it can be useful at multiple places (like counting the number of filters applied)